nixos/modules/services/web-apps/openwebrx.nix

```diff
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.openwebrx;
+in
+{
+  options.services.openwebrx = with lib; {
+    enable = mkEnableOption "Enable OpenWebRX Web interface for Software-Defined Radios";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.openwebrx;
+    };
+
+    user = mkOption {
+      type = types.str;
+      default = "openwebrx";
+    };
+
+    group = mkOption {
+      type = types.str;
+      default = "openwebrx";
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    users = {
+      users.${cfg.user} = {
+        isSystemUser = true;
+        group = cfg.group;
+        home = "/var/lib/openwebrx";
+      };
+      groups.${cfg.group} = {};
+    };
```

Comment:
I would guess that this software can get away with using [DynamicUser](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#DynamicUser=) instead.

Simply remove the `user` and `group` options + definitions, then add `DynamicUser = true;` to `systemd.services.openwebrx.serviceConfig` and you should be good to go.